aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -325,7 +326,7 @@ void upsample_nearest1d_kernel_impl(
     Tensor& output,
     const Tensor& input,
     c10::optional<double> scales_w) {
-  AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::Byte, input.scalar_type(), "upsample_nearest1d", [&] {
+  AT_DISPATCH_FLOATING_TYPES_AND(at::ScalarType::Byte, input.scalar_type(), "upsample_nearest1d", [&] {
```

Comment:
Out of curiosity, "byte" here is for some sort of quantization support?

Reply:
I think the intention is to make `interpolate` be able to accept **uint8** dtype to process raw image, not from the aspect of quantization, same story from [2289](https://github.com/pytorch/vision/issues/2289).
By the way, I didn't add the support for uint8. My original commit only covers floating points. And the request is from a medical equipment company using UNet to process their CT scan images.